### ISSUE TITLE
Explain what SWC is in 'about' page

### ISSUE THREAD

**Issue by karthik** [swcarpentry/DEPRECATED-website#62]
On [the main 'about' page](http://dev.software-carpentry.org/about/index.html) it would be nice if the first sentence described what SWC does instead of skipping directly into other details. For someone with no idea what this group does, it would be nice to know at least a little about the group's mission and larger goals in a sentence or two.

On further consideration is seems to be on the front page. However, I'd still like to learn more about the mission on the about page. But this is just a suggestion (feel free to close this up).

**Comment by gvwilson** [comment]
Please see updated text on home page (about/index.html has been removed). Pull request with better wording would be welcome.

**Comment by karthik** [comment]
This looks fine to me and I don't really have better wording suggestions at this time. If anything, I'd suggest rewording the first sentence as:

>  Software Carpentry helps scientists be more productive by teaching them basic computing skills. 

in large(r) font size. Then the rest of the paragraph can flow as follows:

> Since 1998, we have been running free one- and two-day boot camps at dozens of sites around the world, and also provide open access material online for self-paced instruction. The benefits are more reliable results and higher productivity: a day a week is common, and a ten-fold improvement isn't rare.

But I'll leave it up to you to accept or ignore the change. Perhaps after some period of involvement with SWC, I'll suggest something better.

**Comment by gvwilson** [comment]
Sure --- please go ahead and send a pull request with the change.

**Comment by karthik** [comment]
![image](https://f.cloud.github.com/assets/138494/15245/e32afdd2-4719-11e2-88ca-d6c8b11522cd.png)

**Comment by caitsydney** [comment]
I agree with the need for a first sentence that orients people about what SWC is.

Software Carpentry helps scientists be more productive by teaching them basic computing skills. >>>
After polling the room of scientists (mostly biologists and computer scientists) I'm in, the best intro "sentences" are:
Better code = more science
Better code for more science
Better code for better science
Learning to code for improving science
Software carpentry teaches efficient scientific workflow using computing.
Learning to code to improve scientific workflow
Teaching programming to scientists to improve their workflow
Teaching computing to help scientists improve their workflow

My personal thought is that we should emphasize the part about "teaching computing skills" (or a variation) at the beginning of the sentence/phrase.

**Comment by jiffyclub** [comment]
Agree with @picken19 that we could use a shorter tagline.

Also don't love the hero unit without some balance for all that bold text (images or something), but I think there is a complete refresh of of the home page in the not-too-distant future so not much point worrying about it now.
